docs/ref/contrib/gis/functions.txt

```diff
@@ -346,6 +346,17 @@ resource-intensive) with the ``spheroid`` keyword argument.
     In older versions, a raw value was returned on MySQL when used on
     projected SRS.
 
+``LineLocatePoint``
+===================
+
+.. class:: LineLocatePoint(linestring, point, **extra)
+
+*Availability*: `PostGIS <https://postgis.net/docs/ST_LineLocatePoint.html>`__,
+SpatiaLite
+
+Returns a float between 0 and 1 representing the location of the closest point on
+``linestring`` to the given ``point``, as a fraction of total 2d line length.
```

Comment:
I'm not sure if "total 2d line length" needs a grammar tweak or if I'm not familiar with the terminology. Maybe adding "the total" would help? Possibly 2d should be 2D for consistency with other docs?